Freebox binding 2.0 not working in beta 1

After installing the Freebox binding 2.0 with Paper UI, I created manually a thing (using Paper UI) setting "Freebox Network Address" to 192.168.1.254 and setting "Application token" with my working token I use with openHAB1. When saving I got this exception and the thing remains in a status UNINITIALIZED:

12:19:01.161 [ERROR] [ome.core.thing.internal.ThingManager] - Exception occured while calling thing handler factory 'org.openhab.binding.freebox.internal.FreeboxHandlerFactory@d97cc1': javax/net/ssl/HostnameVerifier
java.lang.NoClassDefFoundError: javax/net/ssl/HostnameVerifier
        at java.lang.ClassLoader.defineClass1(Native Method)[:1.8.0]
        at java.lang.ClassLoader.defineClass(ClassLoader.java:760)[:1.8.0]
        at org.eclipse.osgi.internal.loader.ModuleClassLoader.defineClass(ModuleClassLoader.java:272)
        at org.eclipse.osgi.internal.loader.classpath.ClasspathManager.defineClass(ClasspathManager.java:632)
        at org.eclipse.osgi.internal.loader.classpath.ClasspathManager.findClassImpl(ClasspathManager.java:588)
        at org.eclipse.osgi.internal.loader.classpath.ClasspathManager.findLocalClassImpl(ClasspathManager.java:540)
        at org.eclipse.osgi.internal.loader.classpath.ClasspathManager.findLocalClass(ClasspathManager.java:527)
        at org.eclipse.osgi.internal.loader.ModuleClassLoader.findLocalClass(ModuleClassLoader.java:324)
        at org.eclipse.osgi.internal.loader.BundleLoader.findLocalClass(BundleLoader.java:320)
        at org.eclipse.osgi.internal.loader.BundleLoader.findClassInternal(BundleLoader.java:395)
        at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:345)
        at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:337)
        at org.eclipse.osgi.internal.loader.ModuleClassLoader.loadClass(ModuleClassLoader.java:160)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357)[:1.8.0]
        at org.apache.http.impl.client.HttpClientBuilder.build(HttpClientBuilder.java:709)
        at org.matmaul.freeboxos.internal.RestManager.<init>(RestManager.java:59)
        at org.matmaul.freeboxos.FreeboxOsClient.<init>(FreeboxOsClient.java:52)
        at org.openhab.binding.freebox.handler.FreeboxHandler.authorize(FreeboxHandler.java:97)
        at org.openhab.binding.freebox.handler.FreeboxHandler.initialize(FreeboxHandler.java:145)
        at org.eclipse.smarthome.core.thing.binding.BaseThingHandlerFactory.registerHandler(BaseThingHandlerFactory.java:130)
        at org.eclipse.smarthome.core.thing.internal.ThingManager$6.call(ThingManager.java:507)
        at org.eclipse.smarthome.core.thing.internal.ThingManager$6.call(ThingManager.java:1)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)[:1.8.0]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)[:1.8.0]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)[:1.8.0]
        at java.lang.Thread.run(Thread.java:744)[:1.8.0]
Caused by: java.lang.ClassNotFoundException: javax.net.ssl.HostnameVerifier cannot be found by org.openhab.binding.freebox_2.0.0.b1
        at org.eclipse.osgi.internal.loader.BundleLoader.findClassInternal(BundleLoader.java:432)[org.eclipse.osgi-3.10.2.v20150203-1939.jar:]
        at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:345)[org.eclipse.osgi-3.10.2.v20150203-1939.jar:]
        at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:337)[org.eclipse.osgi-3.10.2.v20150203-1939.jar:]
        at org.eclipse.osgi.internal.loader.ModuleClassLoader.loadClass(ModuleClassLoader.java:160)[org.eclipse.osgi-3.10.2.v20150203-1939.jar:]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357)[:1.8.0]
        ... 26 more

--- a/addons/binding/org.openhab.binding.freebox/META-INF/MANIFEST.MF
+++ b/addons/binding/org.openhab.binding.freebox/META-INF/MANIFEST.MF
@@ -15,6 +15,8 @@ Bundle-ClassPath: .,
  lib/httpcore-4.3.jar
 Import-Package: com.google.common.base,
  com.google.common.collect,
+ javax.net,
+ javax.net.ssl,
  org.apache.commons.logging,
  org.eclipse.smarthome.config.core,
  org.eclipse.smarthome.config.discovery,

